src/Uno.UI/Microsoft/UI/Xaml/Controls/Repeater/ChildrenInTabFocusOrderIterable.cs

```diff
@@ -24,7 +24,7 @@ public ChildrenInTabFocusOrderIterable(ItemsRepeater repeater)
 		private class ChildrenInTabFocusOrderIterator : IEnumerator<DependencyObject>
 		{
 			private readonly List<KeyValuePair<int /* index */, UIElement>> m_realizedChildren;
-			private int m_index = 0;
+			private int m_index = -1;
```

Comment:
They have "C++ iterators" instead of enumerators, so the code is a bit different there, but in C# the enumerator needs to start at -1 to work properly (first call is `MoveNext`)